src/theme/GlobalStyles.ts

```diff
@@ -17,8 +17,8 @@ export const GlobalStyles = createGlobalStyle`
   html {
     font-size: ${(props) => props.theme.fontSizes.baseFontSize};
     font-family: ${(props) => props.theme.fonts.body};;
-    background-color: ${(props) => props.theme.global.bgd};
-    color: ${(props) => props.theme.global.text};
+    background-color: ${(props) => props.theme.globalStyle.bgd};
```

Comment:
This change is not in the CHANGELOG?

Reply:
Do we need a migration guide for this?

Reply:
Updated the CHANGELOG and added a simple migration guide.